docs/references/react/use-sign-up.mdx

```diff
@@ -53,32 +53,49 @@ The `useSignUp()` hook gives you access to the [`SignUp`](/docs/references/javas
 
 ## How to use the `useSignUp()` hook
 
-### Check the current state of a sign-up with `useSignUp()`
+### Check the current state of a sign-up
 
-Use the `useSignUp()` hook to access the `SignUp` object and check the current state of a sign-up.
+The following example uses the `useSignUp()` hook to access the [`SignUp`](/docs/references/javascript/sign-up/sign-up) object and check the current status of a sign-up.
```

Comment:
let's make this match the usesignin() example description

Reply:
https://github.com/clerk/clerk-docs/pull/1814/commits/bce575f331033786c742c87ac7abb911c5f9f128